apps/explorer/src/components/transaction-card/RecentTxCard.tsx

```diff
@@ -119,6 +117,11 @@ export function LatestTxCard({
         parseInt(searchParams.get('p') || '1', 10) || 1
     );
 
+    const handleTabChanged = (tab: number) => {
```

Comment:
Can you instead either move the checkpoints outside of this component or move the pagination into the table?

Reply:
Yea - our designs have the tables in the same `TabGroup` which is how Checkpoints ended up in here. I can re-work this a bit more